Linux/Car_Client/gpio.cpp

```diff
@@ -0,0 +1,138 @@
+#include "gpio.h"
```

Comment:
Looks good. Some function descriptions would be nice (yes, we are now the ones commenting the RC car code ..........)

Reply:
Added comments in GPIO class